### DISCUSSION TITLE
How to get the Galaxy Brain achievement badge

### DISCUSSION THREAD

**Discussion by lincolnvc** [Schweinepriester/github-profile-achievements#41] [Q&A] [answered]
How to get the Galaxy Brain achievement badge?

**Comment by epcgrs** [comment, marked as answer]
Just Write an answer to a question, if you have 2 accepted answers you earn the badge.

**Reply by lincolnvc** [reply]
Thanks

**Comment by GempitaRizki** [comment]
Just write an answer to a question

**Comment by nel0x** [comment]
Not possible anymore https://github.com/Schweinepriester/github-profile-achievements/discussions/45#discussioncomment-3002369
